.github/test_real.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 set -e
+
+cd tests
```

Comment:
Is this the fix to allow it to be installed without link? I thought testflo worked from the repo directory?

Reply:
It's a bit more complicated than that, see the explanation in mdolab/pygeo#99.